Wikipedia/Code/WMFImageTag.m

```diff
@@ -37,16 +37,22 @@ - (instancetype)initWithSrc:(NSString*)src
         self.src = src;
         self.srcset = srcset;
         self.alt = alt;
-        self.width = @([width integerValue]);
-        self.height = @([height integerValue]);
-        self.dataFileWidth = @([dataFileWidth integerValue]);
-        self.dataFileHeight = @([dataFileHeight integerValue]);
+        self.width = [width isEqual:[NSNull null]] ? nil : @([width integerValue]);
+        self.height = [height isEqual:[NSNull null]] ? nil : @([height integerValue]);
+        self.dataFileWidth = [dataFileWidth isEqual:[NSNull null]] ? nil : @([dataFileWidth integerValue]);
+        self.dataFileHeight = [dataFileHeight isEqual:[NSNull null]] ? nil : @([dataFileHeight integerValue]);
     }
     return self;
 }
 
-- (BOOL)isWideEnoughForGallery {
-    return self.width.integerValue > WMFImageTagMinimumWidthForGalleryInclusion && self.dataFileWidth.integerValue > WMFImageTagMinimumWidthForGalleryInclusion;
+- (BOOL)isSizeLargeEnoughForGalleryInclusion {
```

Comment:
I verified that the images I was seeing before that were wide and short (and weren't in the app store version gallery) are no longer in the gallery. Looks good